test/functional/test_site/testTrimIncludeFragment.md

```diff
@@ -0,0 +1,5 @@
+<span id="fragment">
+
+    Fragment with leading spaces and newline
+
+</span>
```

Comment:
Small inconsistency, fragment files have the extension of `.mbdf` instead of `.mb` (given that you explicitly named the file a fragment).